pypiserver/config.py

```diff
@@ -565,13 +597,21 @@ def log_level(self) -> int:
         # If we've specified 3 or more levels of verbosity, just return not set.
         return levels.get(self.verbosity, logging.NOTSET)
 
-    def iter_packages(self) -> t.Iterator[core.PkgFile]:
-        """Iterate over packages in root directories."""
-        yield from (
-            itertools.chain.from_iterable(
-                core.listdir(str(r)) for r in self.roots
-            )
-        )
+    def get_backend(self, arg: str) -> IBackend:
+
+        available_backends: t.Dict[str, BackendFactory] = {
+            "auto": get_file_backend,
+            "simple-dir": SimpleFileBackend,
+            "cached-dir": CachingFileBackend,
+        }
+        # if arg not in available_backends:
+        #     raise argparse.ArgumentTypeError(
+        #         f"Value must be one of {', '.join(available_backends.keys())}"
+        #     )
```

Comment:
Dead code?

Reply:
Ah thanks, yeah I thought I might need to check that here, but it's actually picked up by argparse